### DISCUSSION TITLE
A hashtag to disable ChatGPT responses

### DISCUSSION THREAD

**Discussion by dizcza** [community/community#67157] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

ChatGPT meaningless responses turn a conversation on GitHub into a scam.

Currently, when I say that I don't want any chatgpt responses, it still goes on and on. See https://github.com/ultralytics/ultralytics/issues/4731 for example.

I'd like to turn off any chatgpt responses by adding a hashtag #nochatgpt or similar.

**Comment by dizcza** [comment]
See also 

https://github.com/ultralytics/ultralytics/issues/3220#issuecomment-1629968545

https://github.com/ultralytics/ultralytics/issues/2463#issuecomment-1546631280

> I suspect you're ChatGPT, these codes don't work at all

is my favourite one.

**Comment by ngtanthanh** [comment]
thanks
